lib/svg.dart

```diff
@@ -87,8 +71,6 @@ class SvgPicture extends StatelessWidget {
     this.semanticsLabel,
     this.excludeFromSemantics = false,
     this.theme = const SvgTheme(),
-    @deprecated Clip? clipBehavior,
```

Comment:
Ditto here. I'd like to keep these deprecated properties around for a while to ease migration.

Reply:
Hmm ... I would actually argue that they actually hinder migration, since by keeping them as noop it's not safe to assume that we can "ignore" deprecation warnings as in this case the deprecation already happened.